### DISCUSSION TITLE
chksum blake3 not available on 2.1.5 for linux? i've installed the 2.1.5 but cant use blake3 as checksum as it is missing

### DISCUSSION THREAD

**Discussion by vipsland** [openzfs/zfs#13820] [Q&A] [answered]
chksum blake3 not available on 2.1.5 for linux? i've installed the 2.1.5 but cant use blake3 as checksum as it is missing

**Comment by ghost** [comment, marked as answer]
New features such as this this are unlikely to be backported to the stable release branch.
